crates/bevy_render/src/view/visibility/render_layers.rs

```diff
@@ -20,6 +20,36 @@ pub type Layer = usize;
 /// An entity with this component without any layers is invisible.
 ///
 /// Entities without this component belong to layer `0`.
+///
```

Comment:
I feel like this might almost be too much documentation relative to the feature but the quality here is really good, if anything we should just expand the docs just above this in another PR so it doesn't feel unbalanced.